cttportfolio.html

```diff
@@ -15,14 +15,15 @@
 		</a> 
 		<div class="navlinks icon_hover">
 			<a id="home_link" href="#">Home</a>
-			<a href="#aboutme_jump">About me</a>
-			<a href="#portfolio_jump">Portolio</a>
-			<a href="#contactme_jump">Contact me</a>
+			<a class="navsublinks" href="#aboutme_jump">About me</a>
+			<a class="navsublinks" href="#portfolio_jump">Portolio</a>
+			<a class="navsublinks" href="#contactme_jump">Contact me</a>
+			<a class="nav_dropdown icon_hover" href="#contactme_jump">&#9660;</a>
 		</div>
 	</nav>
 	<header id="home_jump">
-		<video autoplay muted loop preload="metadata" id="frontpagevideo" poster="https://i.imgur.com/fsCfb5b.jpg">
-			<source src="backgroundvideo.mp4" type="video/mp4">
+		<video autoplay muted loop preload="metadata" id="frontpagevideo" class="rotate" poster="https://i.imgur.com/fsCfb5b.jpg">
+			<source src="https://cdn-b-east.streamable.com/video/mp4/ao2ra.mp4?token=Rnr591GaaY5QmsyPfz3V9g&expires=1583366880" type="video/mp4">
 		</video>
```

Comment:
if this isnt working why not add it as an img?

Reply:
it was working but i think theres an issue with streamable where you can only have it used so many times.